bin/docker-build-cni-plugin

```diff
@@ -22,4 +22,4 @@ validate_go_deps_tag "$dockerfile"
     "$bindir"/docker-build-go-deps
 ) >/dev/null
 
-docker_build cni-plugin "$(head_root_tag)" "$dockerfile"
+docker_build cni-plugin $(head_root_tag) "$dockerfile"
```

Comment:
why remove the quotes from head_root_tag?

Reply:
If you look at the head_root_tag function, it will never return whitespace (and if it did, it would not be a valid tag for use with docker build at line 45 in _docker.sh where it would end up).